pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -950,7 +952,16 @@ func appNetworkDoActivateUnderlayNetwork(
 	if err != nil {
 		addError(ctx, status, "createACL", err)
 	}
-	ulStatus.ACLRules = ruleList
+	appID := status.UUIDandVersion.UUID
+	tmpMap := ctx.NLaclMap[appID]
+	if tmpMap == nil {
+		ctx.NLaclMap[appID] = make(map[string]types.ULNetworkACLs)
+	}
+	if _, ok := ctx.NLaclMap[appID][ulStatus.Name]; !ok {
+		ctx.NLaclMap[appID][ulStatus.Name] = types.ULNetworkACLs{}
+	}
+	rlist := types.ULNetworkACLs{ACLRules: ruleList}
+	ctx.NLaclMap[appID][ulStatus.Name] = rlist
```

Comment:
Can we encapsulate this in a setACLRules(appUUID) and also have a getACLRules(appUUID)?

Reply:
ok.